frontend/.eslintrc.js

```diff
@@ -36,8 +36,6 @@ module.exports = {
     "plugin:prettier/recommended",
     // Recommended Jest configuration to enforce good testing practices
     "plugin:jest/recommended",
-    // Uses the recommended rules from React Testing Library:
-    "plugin:testing-library/react",
```

Comment:
**note:** Moved to a scoped section so that these rules only apply on test files to prevent false negatives